.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "spdk"]
 	path = spdk
 	url = https://github.com/ceph/spdk.git
-	branch = ceph-nvmeof-v24.01
+	branch = leonid-ceph-nvmeof-v24.05
```

Comment:
It seems this PR includes an unintended SPDK change.

Reply:
what exactly? , I wanted to add Async event notification per ns-change lb group rpc